### DISCUSSION TITLE
Mounting using a config file and SAS token

### DISCUSSION THREAD

**Discussion by sbillsbo** [Azure/azure-storage-fuse#918] [Q&A] [answered]
We are trying to mount using a SAS token so that we can provide read only access to the blob container.  Here is the error message we are getting:

Unable to start blobfuse due to a lack of credentials. Please check the readme for valid auth setups

**Comment by francoisdutoit** [comment]
@sbillsbo Have you manged to resolve your issue? Maybe I can assist.

**Comment by bhansalivikas** [comment, marked as answer]
SAS need to have execute permission to list directories. Just having read access may not work. Kindly check the SAS permissions and verify.

**Comment by sbillsbo** [comment]
We decided to go another way. We used a token with Read write and used chmod 444 on the directory where the share is mounted to make it read only.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by bhansalivikas** [comment]
So its working for you now or you are still facing some issue with access?

**Comment by sbillsbo** [comment]
It is working fine. Thanks.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
